Assignment2/alvaroalvi-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,69 @@
+{
+	"@context":{
+		"xsd":"http://www.w3.org/2001/XMLSchema#",
+		"pf":"http://www.ontologies.org/ontologies/Class#",
+		"includes": "rdfs:label",
+		"hasMeasurement": "rdfs:label",
+		"hasTemperature":{
+			"rdfs":"label",
+			"@type":"xsd:date"
+		},
+		"hasOwner":"rdfs:label",
+		"atTime":{
+			"rdfs":"label",
+			"@type":"xsd:dateTime"
+		},
+		"hasName":{
+			"rdfs":"label",
+			"@type":"xsd:string"
+		}
+	},
+	
+	"pf:Class01":[
+		{
+			"includes":[
+				{
+					"pf:Sensor029":[

```

Comment:
You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes.